medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,257 @@
+
+        for mode in search_strings:
+            items = []
+            log.debug(u'Search Mode: {0}'.format(mode))
+
+            # if looking for season, look for more pages
+            if mode == 'Season' and not manual_search:
+                self.max_back_pages = 10
+
+            for search_string in search_strings[mode]:
+                if mode != 'RSS':
+                    log.debug(u'Search string: {0}'.format(search_string.decode('utf-8')))
+
+                # Remove season / episode from search (not supported by tracker)
+                search_str = re.sub(r'\d+$' if anime else r'[S|E]\d\d', '', search_string).strip()
+                search_params['searchstr'] = search_str
+                next_page = 1
+                has_next_page = True
+
+                while has_next_page and next_page <= self.max_back_pages:
+                    search_params['page'] = next_page
+                    log.debug(u'Page Search: {0}'.format(next_page))
+                    next_page += 1
+
+                    response = self.session.get(self.urls['search'], params=search_params)
+                    if not response:
+                        log.debug('No data returned from provider')
+                        continue
+
+                    result = self._parse(response.content, mode)
+                    has_next_page = result['has_next_page']
+                    items += result['items']
+
+                results += items
+
+        return results
+
+    def _parse(self, data, mode):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+
+        :return: A KV with a list of items found and if there's an next page to search
+        """
+        def process_column_header(td):
+            ret = u''
+            if td.a and td.a.img:
+                ret = td.a.img.get('title', td.a.get_text(strip=True))
+            if not ret:
+                ret = td.get_text(strip=True)
+            return ret
+
+        items = []
+        has_next_page = False
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find('table', id='torrent_table')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # ignore next page in RSS mode
+            has_next_page = mode != 'RSS' and html.find('a', class_='pager_next') is not None
+            log.debug(u'More Pages? {0}'.format(has_next_page))
+
+            # Continue only if at least one Release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return {'has_next_page': has_next_page, 'items': []}
+
+            # '', '', 'Name /Year', 'Files', 'Time', 'Size', 'Snatches', 'Seeders', 'Leechers'
+            labels = [process_column_header(label) for label in torrent_rows[0]('td')]
+            group_title = u''
```

Comment:
The `u` prefix needs to go.

Reply:
I'm not python developer at daily but that field needs to be unicode, there's no problem if i init it as str and change to unicode later?

Reply:
Done, changed to None